File: TaskManager/DAL/Model/LoginResult.cs
using DAL.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Model
{
    public class LoginResult
    {
        public string userId { get; set; }
        public string username { get; set; }
        public string firstname { get; set; }
        public string lastname { get; set; }
        public string mobile { get; set; }
        public string token { get; set; }

    }
}

File: ui/src/_models/login-result.ts
export class LoginResult {
  username: string;
  firstname: string;
  lastname: string;
  mobile:string;
  userId: string;
  token: string;
  constructor() {
    this.token = '';
    this.username = '';
    this.firstname = '';
    this.lastname = '';
    this.mobile = '';
    this.userId = '';
  }
}
